ctx_test.go

```
@@ -3618,6 +3620,111 @@ func Benchmark_Ctx_JSON(b *testing.B) {
+
+	type SomeStruct struct {
+		Name string `cbor:"name"`
+		Pass string `cbor:"pass"`
+	}
+
+	data := SomeStruct{
+		Name: "john",
+		Pass: "doe",
+	}
+
+	a, _ := cbor.Marshal(data)
```

golangci-lint (1.62.2)

Error return value of `cbor.Marshal` is not checked

(errcheck)